azure-kusto-ingest/tests/sample.py

```diff
@@ -10,12 +10,13 @@
     FileDescriptor,
     BlobDescriptor,
     StreamDescriptor,
-    DataFormat,
     KustoStreamingIngestClient,
     ManagedStreamingIngestClient,
     IngestionStatus,
 )
 
+from azure.kusto.data.data_format import DataFormat
```

Comment:
Put it next to the other azure.kusto.data imports

In fact, shouldn't we just export it from azure.kusto.data directly?